### PR TITLE
add url_suffix for custom javascript files

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
# What this PR does

This PR adds url_suffix to all ome custom JavaScript files

# Testing this PR

To test this PR go to:
 - page webclient/userdata/ check:
    - `<script type="text/javascript" src="{% static "webclient/javascript/ome.right_panel_tags_*` 
    - `<script type="text/javascript" src="{% static "webclient/javascript/jquery.jstree.*`
 - page webclient/public/ check:
   - `<script type="text/javascript" src="{% static "webclient/javascript/jquery.jstree.*`
has build number

Basically, make sure if all JS files contains either build number or 3rd party libraries container version number in the path

cc @will-moore

### CODE REVIEW

**Comment by joshmoore** [comment]
> Basically, make sure if all JS files contains either build number or 3rd party libraries container version number in the path

@aleksandra-tarkowska : is this something we need to teach travis?

**Comment by aleksandra-tarkowska** [comment]
@joshmoore that may be hard to do, but we could certainly try. I think it will be easy to go around them.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-DEV-merge-push#362](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/362/). See the [console output](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-push/362/consoleText) for more details.
Possible conflicts:
  - PR #4630 will-moore 'Open with'
    - components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/base/base_container.html
  - PR #4682 aleksandra-tarkowska 'remove rank and sort by name regardless of node type (rebased onto develop)'
    - components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/base/base_container.html
  - PR #4703 will-moore 'histograms'
    - components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/base/base_container.html

**Comment by joshmoore** [comment]
> @joshmoore that may be hard to do, but we could certainly try. 

A starting point:
```
$ git grep -E 'script\stype="text/javascript" src="[{]% static "webclient/javascript/ome' -- components/tools/OmeroWeb/omeroweb/webclient/templates
```

**Comment by aleksandra-tarkowska** [comment]
as you can see https://github.com/openmicroscopy/openmicroscopy/pull/4728/files#diff-e909c1f1ca2d62131dddb3e56bb0c434R41 they are missing ome prefix

**Comment by aleksandra-tarkowska** [comment]
I am guessing the easiedt would be if no digits then must be prefixed (this rule will then be able to apply to all even 3rd party

The origianl policy was to have number in the path for 3rd party libs and `url_suffix` for all custom written/ome related

**Comment by will-moore** [comment]
Changes look good.
A quick check of network requests for js files - didn't see any that were missing version or the url_suffix.
Good to merge.

**Comment by aleksandra-tarkowska** [comment]
@joshmoore I have basic flake8 parser for that, although I am still missing `noqa` implementation and tests. I will try to open a pr soon.

**Comment by aleksandra-tarkowska** [comment]
@joshmoore I have basic parser for templates, although I am still missing how to put it as flake8 plugin

**Comment by joshmoore** [comment]
> I have basic parser for templates, although I am still missing how to put it as flake8 plugin. these are no py files

Not sure if we should try to work non-py files into flake8. Might be better to just add a separate call to a script under `tools/`, `tools/OmeroWeb`, etc. into travis.

**Comment by aleksandra-tarkowska** [comment]
that was my impression, how about that?

**Comment by aleksandra-tarkowska** [comment]
failing as expected in https://travis-ci.org/openmicroscopy/openmicroscopy/jobs/142736227

**Comment by aleksandra-tarkowska** [comment]
failed as expected https://travis-ci.org/openmicroscopy/openmicroscopy/jobs/142744839

**Comment by aleksandra-tarkowska** [comment]
https://travis-ci.org/openmicroscopy/openmicroscopy/jobs/142772919#L537 is passing as well as test/unit/test_validator.py

**Comment by aleksandra-tarkowska** [comment]
I moved Travis cehck to another pr, to not block it
